### PR TITLE
Added Validation for confstore cluster config keys in utils_setup

### PR DESCRIPTION
**Pull request by rahul-tripathi-git**
Signed-off-by: Rahul Tripathi <rahul.tripathi@seagate.com>

### CODE REVIEW

**Comment by cortx-admin** [comment]
Can one of the admins verify this patch?

**Comment by rahul-tripathi-git** [comment]
run mini-provisioning

File: py-utils/src/setup/utils.py [hunk 1]
```diff
@@ -19,13 +19,14 @@
...
 from cortx.utils.process import SimpleProcess
-
+from cortx.utils.validator.v_confkeys import ConfKeysV
 
 def _get_kafka_server_list(conf_url):
     """ Reads the ConfStore and derives keys related to message bus """
 
     from cortx.utils.conf_store import Conf
     Conf.load("cluster_config", conf_url)
+    ConfKeysV.validate(v_type="exists", index="cluster_config", keylist=["cortx>software>common>message_bus_type"])
```
Comment (ujjwalpl) [inline py-utils/src/setup/utils.py:29]: keylist needs to include all the keys. 
```
keylist = [ 'cortx>software>common>message_bus_type', 'cortx>software>common>message_servers' ]
ConfKeysV.validate("exists", "cluster_config", keylist)
```
Also, take care of indentation and line exceeding 80 chars.

**Review by ujjwalpl — CHANGES_REQUESTED** [review]

File: py-utils/src/setup/utils.py (hunk 1 above)
Comment (rahul-tripathi-git) [reply py-utils/src/setup/utils.py]: Thanks for pointing these out @ujjwalpl , fixed both, and tested locally.

File: py-utils/src/setup/utils.py [hunk 2]
```diff
@@ -19,13 +19,18 @@
...
 from cortx.utils.process import SimpleProcess
-
+from cortx.utils.validator.v_confkeys import ConfKeysV
 
 def _get_kafka_server_list(conf_url):
     """ Reads the ConfStore and derives keys related to message bus """
 
     from cortx.utils.conf_store import Conf
     Conf.load("cluster_config", conf_url)
+
```
Comment (ujjwalpl) [inline py-utils/src/setup/utils.py:29]: Please move global functions to within the class.

**Review by ujjwalpl — CHANGES_REQUESTED** [review]

File: py-utils/src/setup/utils.py (hunk 2 above)
Comment (rahul-tripathi-git) [reply py-utils/src/setup/utils.py]: Thanks for pointing this out @ujjwalpl, made the suggested changes.

**Review by ujjwalpl — APPROVED** [review]

**Review by rahul-telawade — APPROVED** [review]
Look ok for me

**Comment by rahul-tripathi-git** [comment]
run mini-provisioning

**Review by suryakumar1024 — APPROVED** [review]

**Pull request merged**